pkg/analysis/completion.go

```diff
@@ -123,6 +123,14 @@ func (a *Analyzer) completeExpression(doc document.Document, nodes []*sitter.Nod
 		}
 	}
 
+	if len(symbols) == 0 {
```

Comment:
1. `if len(symbols) == 0` seems like a misleading condition here - whether we complete to members shouldn't depend on whether there are other completion options, but rather whether we're following a `.`, right?
2. I tried testing this by typing `nonexistentidentifier.` and the completion options were all members of all builtins

Reply:
Yeah, maybe. I'm not sure what the right behavior should be. We could probably take this away and it would still work pretty well. There are some other cases like `os.path` etc. where you'd get methods in the completion list.